lms/djangoapps/discussion_api/serializers.py

```diff
@@ -75,11 +77,20 @@ def validate_not_blank(value):
         raise ValidationError("This field may not be blank.")
 
 
+def get_user_profile_image_url(request, username):
+    """
+    Returns image urls against a user profile
+    """
```

Comment:
It would be nice if we can start formatting the doc strings like this:
```
def func(arg1, arg2):
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
```
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings